tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_16.py

```diff
@@ -127,3 +127,38 @@ def test_raise_error_if_zero_f_bpt(self) -> None:
                 sigma_pm0=sigma_pm0,
                 f_bpt=f_bpt,
             )
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (
+                    r"l_{pt} = \alpha_1 \cdot \alpha_2 \cdot \Phi \cdot \frac{\sigma_{pm0}}{f_{bpt}} = "
+                    r"1.00 \cdot 0.25 \cdot 8.00 \cdot \frac{350.00}{5.00} = 140.00"
```

Comment:
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/c7e9d117-75c5-46a6-9ceb-1dd098b96db5)
```suggestion
                    r"l_{pt} = \alpha_1 \cdot \alpha_2 \cdot Ø \cdot \frac{\sigma_{pm0}}{f_{bpt}} = "
                    r"1.00 \cdot 0.25 \cdot 8.00 \cdot \frac{350.00}{5.00} = 140.00"
```